app/views/users/_form.html.erb

```diff
@@ -26,24 +26,24 @@
   <div class="form-row row">
     <div class="column small-12 medium-5 medium-offset-1">
       <h4 class="c-title">Position</h4>
-      <%= f.input :position, label: false %>
+      <%= f.input :position, placeholder: "e.g., Director", label: false %>
     </div>
 
     <div class="column small-12 medium-5">
       <h4 class="c-title">Pubmed</h4>
-      <%= f.input :pubmed, placeholder: "", label: false %>
+      <%= f.input :pubmed, placeholder: "Name Last Name", label: false %>
     </div>
   </div>
 
   <div class="form-row row">
     <div class="column small-12 medium-5 medium-offset-1">
       <h4 class="c-title">Twitter</h4>
-      <%= f.input :twitter_account, placeholder: "@mbarrenech", label: false %>
+      <%= f.input :twitter_account, placeholder: "User name", label: false %>
```

Comment:
As long as we need to insert `"@username"` let's use this instead of `"User name"`